api/v2/v2.go

```diff
@@ -0,0 +1,190 @@
+// Copyright (c) 2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package v2
+
+import (
+	"regexp"
+
+	"github.com/decred/dcrtime/merkle"
+)
+
+type ResultT int
+
+const (
+	APIVersion = 2
+
+	// VersionRoute defines a top-level API route for retrieving latest version
+	VersionRoute = "/version/"
```

Comment:
imo the version route should be in v1 and should be defined as `/v1/version`, not just `/version`, but marco said something different in the chat so check with him on this.

Reply:
Marco mentioned this should be a top-level route that isn't versioned. Still, wasn't sure where to properly define it, since v2 is versioned so doesn't seem right. Can it be defined in the daemon directly?

link to discussion: https://matrix.to/#/!gltiHJRZiSJTzvjOEu:decred.org/$158100832514045xveVx:decred.org?via=decred.org

Reply:
No it either needs to be in v1 or v2.  Check with marco where he wants it.

Reply:
I did indeed tell @thi4go to put it unversioned. What it should return is the prefix for the v1 and v2 API. Then the caller can pick which one to use.

We should add a config option that enables v1, v2 or both.